Client/src/Assets/emptyJSFile.json

```diff
@@ -0,0 +1,96 @@
+{
+    "reference": {
+        "type": "",
+        "publisher": "",
+        "authors": [
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            },
+            {
+                "firstname": "",
+                "middlename": "",
+                "lastname": ""
+            }
+        ],
+        "title": "",
+        "volume": 0,
+        "pages": 0,
+        "year": 0
```

Comment:
After year there should be these two fields:
"datePublished": ""  
"dateAccessed": ""

Reply:
done